src/vivarium/framework/randomness/stream.py

```diff
@@ -316,12 +321,10 @@ def random(
     return pd.Series(index=index, dtype=float)  # Structured null value
 
 
-def choice(
-    key: str,
-    index: Union[pd.Index, pd.MultiIndex],
+def _choice(
```

Comment:
Why does this method even need to exist outside the randomness stream if we're making it private?

Reply:
+1

Reply:
It does not.  It's a more complicated change and will be addressed later.